test/services/ConsignmentServiceSpec.scala

```diff
@@ -435,4 +442,40 @@ class ConsignmentServiceSpec extends AnyWordSpec with MockitoSugar with BeforeAn
       results.failed.futureValue shouldBe a[HttpError]
     }
   }
+
+  "getConsignmentsHistory" should {
+    "return list of consignments for the user" in {
```

Comment:
Maybe reword the test title slightly to: `return a list of the user's consignments`